content/update-pr.js

```diff
@@ -1,7 +1,11 @@
 const _ = require('lodash')
 const md = require('./template')
 
-module.exports = ({version, dependencyLink, dependency, monorepoGroupName, release, diffCommits, oldVersionResolved, type}) => md`
+module.exports = ({version, dependencyLink, dependency, monorepoGroupName, release, diffCommits, oldVersionResolved, type, yearlyBillingAd}) => md`
+${yearlyBillingAd
+    ? `👛 Greenkeeper now offers yearly billing options. Get two month for free!\n`
```

Comment:
Suggestion: `👛 Save 16% on your Greenkeeper invoice by switching to yearly billing in the [account dashboard](billingURL).`, where `billingURL` is `https://account.greenkeeper.io/account/${ORGNAME}#plans-and-billing`. That would require passing in the username/orgname from `create-version-branch`, which should just be `repository.fullName.split('/')[0]`.